prelexer.cpp

```diff
@@ -423,6 +423,9 @@ namespace Sass {
     const char* number(const char* src) {
       return sequence< optional<sign>, unsigned_number>(src);
     }
+    const char* sass_number(const char* src) {
+      return sequence< optional<sign>, unsigned_number, optional < identifier > >(src);
```

Comment:
My tests seems to indicate that ruby sass allows pretty funny units for numbers, so yes, it should IMO be quite correct. But I think it might be usefull to explicitly define a prelexer for number units. Basically its the same as number + units. This fixes following case:

```
1px    /     2px
```